_includes/tutorials/rekeying-function/ksql/markup/dev/make-udf.adoc

```diff
@@ -0,0 +1,7 @@
+Then create the following file at `src/main/java/io/confluent/developer/RegexReplace.java`.
```

Comment:
Could use some more explanation here about what this is and what it does. This is really key (pun intended!) to the whole thing.

Reply:
Could also be good to talk about why we're handling nulls, and when you should watch out for them.

Reply:
Done